serverless/integrations/serverless-cost-management-integration.adoc

```diff
@@ -0,0 +1,24 @@
+:_content-type: ASSEMBLY
+[id="serverless-cost-management-integration"]
+= Integrating {ServerlessProductShortName} with the cost management service
+include::_attributes/common-attributes.adoc[]
+:context: serverless-cost-management-integration
+
+toc::[]
+
+link:https://access.redhat.com/documentation/en-us/cost_management_service/2022/html/getting_started_with_cost_management/assembly-introduction-cost-management#about-cost-management_getting-started[Cost management] is an {product-title} service that enables you to better understand and track costs for clouds and containers. It is based on the open source link:https://project-koku.github.io/[Koku] project.
+
+[id="prerequisites_serverless-cost-management-integration"]
+== Prerequisites
+
+* You have cluster administrator permissions.
+
+* You have set up cost management and added an link:https://access.redhat.com/documentation/en-us/cost_management_service/2022/html/adding_an_openshift_container_platform_source_to_cost_management/index[{product-title} source].
+
+include::modules/serverless-cost-management-labels.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+[id="additional-resources_serverless-cost-management-integration"]
+== Addiitonal resources
+* link:https://access.redhat.com/documentation/en-us/cost_management_service/2022/html/getting_started_with_cost_management/assembly-installing-cost-management#configure-tagging-next-step_configuring[Configure tagging for your sources]
```

Comment:
Should these state that they are in the cost management documentation, just so it's not a surprise to be redirected away (especially when coming from docs.openshift.com)?

Reply:
It's part of the [Jupiter checklist requirements](https://docs.engineering.redhat.com/pages/viewpage.action?spaceKey=JUP&title=Jupiter+Source+files+readiness+Checklist#JupiterSourcefilesreadinessChecklist-Additionalresourcesspecificrequirements) to just have the links with no plain text for additional resources, so that's why I added them this way.

Reply:
Fair enough then!